### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Rémy Bleuer

### DISCUSSION THREAD

**Discussion by RemyBlr** [heig-vd-dai-course/heig-vd-dai-course#130] [Show and tell]
https://github.com/RemyBlr/dai-bleuer-practical-work-10

<img width="465" alt="Capture d’écran 2023-10-25 à 09 59 54" src="https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/35931021/59c50aad-ea57-44f6-bbdf-756afce66eb3">

**Comment by ludelafo** [comment]
Everything looks good to me!
